### ISSUE TITLE
Unable to get trophies using TrophyDataRequest

### ISSUE THREAD

**Issue by danielvetromilla** [ilendemli/gumer-psn-php#18]
I'm getting the following exception everytime i use the **TrophyDataRequest** like the examples to get my PSN trophies.

`Client error response [status code] 400 [reason phrase] Bad Request [url] https://GB-tpy.np.community.playstation.net/trophy/v1/trophyTitles?fields=%40default&npLanguage=EN&iconSize=64&platform=PS3&offset=0&limit=20&comparedUser=cucetilde`

The code:

```
... login code, getting some data, bla bla bla ...

// Get the trophies
try {
    $request    = new Gumer\PSN\Requests\TrophyDataRequest;
    $request->setUserId($info['onlineId']);
    $response   = $connection->call($request);
    $trophy     = json_decode($response->getBody(true), true);

    echo "<pre>";
    var_dump( $trophy );
    echo "</pre>";          
} catch ( Exception $e ) {
	echo $e->getMessage();
}
```

**Comment by qligier** [comment]
Hello,

It's fixed with PR #19.

Quentin

**Comment by danielvetromilla** [comment]
Your fix from commit [8468b7b5940844de59772f2a77f91766e4946540](https://github.com/qligier/gumer-psn-php/commit/8468b7b5940844de59772f2a77f91766e4946540) did the trick

Many thanks @qligier
